Makefile

```diff
@@ -1,3 +1,4 @@
-all    :; DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=200 forge build --use solc:0.8.14
-clean  :; dapp clean
-test   :; make && ./test.sh $(match)
+all          :; DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=200 forge build --use solc:0.8.14
+clean        :; dapp clean
+test         :; make && ./test.sh $(match)
+certora-base :; $(if $(CERTORAKEY),, @echo "set certora key"; exit 1;) PATH=~/.solc-select/artifacts/solc-0.8.14:~/.solc-select/artifacts/solc-0.6.12:~/.solc-select/artifacts/solc-0.5.12:~/.solc-select/artifacts:${PATH} certoraRun --solc_map KilnMock=solc-0.8.14,Dai=solc-0.5.12,Gem=solc-0.6.12,PoolMock=solc-0.6.12 --optimize_map KilnMock=200,Dai=0,Gem=200,PoolMock=0 certora/KilnMock.sol certora/Dai.sol certora/Gem.sol certora/PoolMock.sol --link KilnMock:sell=Dai KilnMock:buy=Gem PoolMock:dai=Dai PoolMock:gem=Gem KilnMock:pool=PoolMock --verify KilnMock:certora/KilnBase.spec --rule_sanity $(if $(rule),--rule $(rule),) --multi_assert_check --short_output
```

Comment:
Is there any way to refactor all of this into a json or other config file and pass it in here?

Reply:
we could move it into a script, following previous patterns.

Reply:
I asked to certora team and it seems that a config file is auto-generated at each run inside the `.certora_internal` run folders. I grabbed the latest and moved it to `.certora.conf` to be passed in the Makefile script instead of the long list of options and config there.
There is still pending the `solc-select` thing, discussed with @kmbarry1 that we could research more to see if possible to add it as well in the config and eventually be removed altogether.
Done in 3134052.

Reply:
I've tried to also cleanup the `solc-select` part in the script path but it seems there is not a better way a the moment, relying on config options or env variables to ensure portability.
A future approach could be via a docker solution, at least for the CI.